.github/workflows/main.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: "2.7.0"
+        ruby-version: "3.3"
```

Comment:
(unsure if this was deliberately set to an old Ruby?)

It was failing with:

```
 ERROR:  Error installing bundler:
  	There are no versions of bundler (= 2.5.5) compatible with your Ruby & RubyGems
  	bundler requires Ruby version >= 3.0.0. The current ruby version is 2.7.0.0.
```

Reply:
This is very intentional, it needs to be able to run with BASERUBY, which is the ruby that is compiling the current Ruby. This cannot change.